inception/config.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import Field
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    transformer_model_name: str = Field(
+        "sentence-transformers/multi-qa-mpnet-base-dot-v1",
+        description="Name of the transformer model to use",
+    )
+    max_words: int = Field(
+        350, ge=1, le=1000, description="Maximum words per chunk"
+    )
+    min_text_length: int = 1
+    max_query_length: int = 100
+    max_text_length: int = 10_000_000
+    max_batch_size: int = 100
+    processing_batch_size: int = 8
+    pool_timeout: int = (
+        3600  # Timeout for multi-process pool operations (seconds)
+    )
+    force_cpu: bool = False
```

Comment:
I didn't understand this one in the readme. Was hoping for more clues here. Maybe tweak the readme to make it more clear?

Reply:
Got it. I've improved the description in the README to:
`Forces the service to run on CPU even if a GPU is available. Useful for debugging or ensuring CPU is selected on query embedding service instances.`

Does it look better now?

I’ve also added all the missing settings to the README and enhanced their descriptions.